### ISSUE TITLE
sound is very low

### ISSUE THREAD

**Issue by daspliff** [nnutter/ubuntu-samus#2]
Hey. I followed your guide and installed ubuntu 15.10 on my pixel. Thanks a million!!. It is so great. I have 2 small issues. The major one is the sound is very low on the system. Any tweaks for that?

Secondly, i applied the touchpad script which works fine, however I notice my touchpad has a right-click zone, which is driving me crazy. Any ideas how to remove it through the .conf file? I already tried placing :
Option   "Button Right Click Enable"  "0" 
in the 50-touchpad-cmt-samus.conf file
This did not work....which makes me wonder if i am editing the right .conf

Thanks for your work

**Comment by nnutter** [comment]
Hi @daspliff, I'd recommend asking about the touchpad on [hugegreenbug/xf86-input-cmt](https://github.com/hugegreenbug/xf86-input-cmt) and the sound on [raphael/linux-samus](https://github.com/raphael/linux-samus).  Those are the corresponding projects that actually do the hard work.  I just documented how I managed to get Ubuntu installed when the installers weren't working.  I'm glad you got that far, good luck!

(For comparison, I do not have a volume problem on mine.)
